packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -205,12 +205,24 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 		);
 	}
 
+	const previewDesign = () => {
+		if ( design.is_virtual && design.style_variation ) {
+			return onPreview( design, design.style_variation );
```

Comment:
There is an instance of `onPreview` that only accepts one parameter:

https://github.com/Automattic/wp-calypso/blob/babd265c6114418f9b59413da06ea57905574ec0/client/signup/steps/design-picker/index.jsx#L164-L176

I'm not quite sure how to get there (my head spins looking at all these onboarding flows!) but we should ensure we don't risk regressions.

Reply:
I think that's a different function. The one here also accepts a 2nd parameter:

https://github.com/Automattic/wp-calypso/blob/4d70821f6e85a329cbb058606b6e91c51d3a9b08/packages/design-picker/src/components/unified-design-picker.tsx#L147

https://github.com/Automattic/wp-calypso/blob/4d70821f6e85a329cbb058606b6e91c51d3a9b08/packages/design-picker/src/components/unified-design-picker.tsx#L233